### PR TITLE
docs: Add `Table` section sample

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

**Comment by plagoa** [comment]
The Applitools test failure is expected as I slightly reduced the width of the `Title` column to keep things tidier.

File: packages/core/src/components/Table/stories/TableSamples/Section.tsx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    borderBottom: `1px solid ${theme.colors.atmo3}`,
+  }),
+  sectionActions: css({
+    right: theme.space.sm,
+  }),
+  pagination: css({
+    margin: 0,
+    backgroundColor: theme.colors.atmo2,
+    padding: theme.space.xs,
+    borderRadius: `0 ${theme.radii.round} ${theme.radii.round} 0`,
```
Comment (MEsteves22) [inline packages/core/src/components/Table/stories/TableSamples/Section.tsx:50]: The left bottom border radius is missing. It should be: `0 0 ${theme.radii.round} ${theme.radii.round}`

**Review by MEsteves22 — COMMENTED** [review]
Looks good 👌🏻

I just noticed the following:

File: packages/core/src/components/Table/stories/TableSamples/Section.tsx (hunk 1 above)
Comment (plagoa) [reply packages/core/src/components/Table/stories/TableSamples/Section.tsx]: fixed

**Review by MEsteves22 — APPROVED** [review]

**Pull request merged**